clang/test/SemaSYCL/native_cpu/no-opt.cpp

```diff
@@ -0,0 +1,22 @@
+// The sycl-native-cpu helper header is always named <sycl-int-header>.hc
+// RUN: %clangxx -fsycl -fsycl-native-cpu -O0 -o %t.bc %s 
```

Comment:
Doesn't seem like a `Sema` test, should probably be moved to integration tests, if we expect some passes to be launched here.

Reply:
Moved the test, thank you